dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -967,6 +968,19 @@ If you want to disable this behaviour, set the env **CLEAN_LOCAL_TAGS** to false
 ./dev/provider_packages/tag_providers.sh
 ```
 
+## Update providers metadata
+
+```shell script
+branch="update-providers-metadata-$(date '+%Y-%m-%d%n')
+git checkout -b "${branch}"
+breeze release-management generate-providers-metadata
+git add -p ,
+git commit -m "Update providers metadata $(date '+%Y-%m-%d%n')"
```

Comment:
This may induce a bug if you do this right around midnight 😆 Maybe put the date string in a variable instead?